### ISSUE TITLE
1.0.1 version, Building Error - ERROR in DataTableModule is not an NgModule

### ISSUE THREAD

**Issue by lar-ang** [jdcalkins/ng2-data-table#22]
I updated version 1.0.1, and while building it's error.

**`ERROR in DataTableModule is not an NgModule`**

And this is installing log.

```
D:\Work\Project\TripperBus\TripperBus\FrontEnd>npm install ng2-data-table --verbose -S
npm info it worked if it ends with ok
npm verb cli [ 'C:\\Program Files\\nodejs\\node.exe',
npm verb cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
npm verb cli   'install',
npm verb cli   'ng2-data-table',
npm verb cli   '--verbose',
npm verb cli   '-S' ]
npm info using npm@3.10.10
npm info using node@v6.10.0
npm verb request uri https://registry.npmjs.org/ng2-data-table
npm verb request no auth needed
npm info attempt registry request try #1 at 4:53:20 PM
npm verb request id 69f55a50de23fc12
npm verb etag W/"592f9777-8908"
npm verb lastModified Thu, 01 Jun 2017 04:26:31 GMT
npm http request GET https://registry.npmjs.org/ng2-data-table
npm http 304 https://registry.npmjs.org/ng2-data-table
npm verb headers { date: 'Thu, 01 Jun 2017 08:53:40 GMT',
npm verb headers   via: '1.1 varnish',
npm verb headers   'cache-control': 'max-age=300',
npm verb headers   etag: 'W/"592f9777-8908"',
npm verb headers   age: '0',
npm verb headers   connection: 'close',
npm verb headers   'x-served-by': 'cache-lax8638-LAX',
npm verb headers   'x-cache': 'HIT',
npm verb headers   'x-cache-hits': '1',
npm verb headers   'x-timer': 'S1496307220.112668,VS0,VE157',
npm verb headers   vary: 'Accept-Encoding' }
npm verb etag https://registry.npmjs.org/ng2-data-table from cache
npm verb get saving ng2-data-table to C:\Users\jonhs\AppData\Roaming\npm-cache\registry.npmjs.org\ng2-data-table\.cache.json
npm verb correctMkdir C:\Users\jonhs\AppData\Roaming\npm-cache correctMkdir not in flight; initializing
npm verb cache add spec ng2-data-table@^1.0.1
npm verb addNamed ">=1.0.1 <2.0.0" is a valid semver range for ng2-data-table
npm verb addNameRange registry:https://registry.npmjs.org/ng2-data-table not in flight; fetching
npm verb get https://registry.npmjs.org/ng2-data-table not expired, no request
npm verb addNamed "1.0.1" is a plain semver version for ng2-data-table
npm verb afterAdd C:\Users\jonhs\AppData\Roaming\npm-cache\ng2-data-table\1.0.1\package\package.json not in flight; writing
npm verb correctMkdir C:\Users\jonhs\AppData\Roaming\npm-cache correctMkdir not in flight; initializing
npm verb afterAdd C:\Users\jonhs\AppData\Roaming\npm-cache\ng2-data-table\1.0.1\package\package.json written
npm verb request uri https://registry.npmjs.org/fsevents
npm verb request no auth needed
npm info attempt registry request try #1 at 4:53:26 PM
npm verb etag W/"5924f6b0-d606"
npm verb lastModified Wed, 24 May 2017 02:57:52 GMT
npm http request GET https://registry.npmjs.org/fsevents
npm http 304 https://registry.npmjs.org/fsevents
npm verb headers { date: 'Thu, 01 Jun 2017 08:53:44 GMT',
npm verb headers   via: '1.1 varnish',
npm verb headers   'cache-control': 'max-age=300',
npm verb headers   etag: 'W/"5924f6b0-d606"',
npm verb headers   age: '269',
npm verb headers   connection: 'close',
npm verb headers   'x-served-by': 'cache-lax8626-LAX',
npm verb headers   'x-cache': 'HIT',
npm verb headers   'x-cache-hits': '2',
npm verb headers   'x-timer': 'S1496307225.783918,VS0,VE0',
npm verb headers   vary: 'Accept-Encoding' }
npm verb etag https://registry.npmjs.org/fsevents from cache
npm verb get saving fsevents to C:\Users\jonhs\AppData\Roaming\npm-cache\registry.npmjs.org\fsevents\.cache.json
npm verb correctMkdir C:\Users\jonhs\AppData\Roaming\npm-cache correctMkdir not in flight; initializing
npm verb correctMkdir C:\Users\jonhs\AppData\Roaming\npm-cache\_locks correctMkdir not in flight; initializing
npm verb lock using C:\Users\jonhs\AppData\Roaming\npm-cache\_locks\staging-707e5d5003dbcf66.lock for D:\Work\Project\TripperBus\TripperBus\FrontEnd\node_modules\.staging
npm verb unbuild node_modules\.staging\ng2-data-table-f8e5bea5
npm verb gentlyRm don't care about contents; nuking D:\Work\Project\TripperBus\TripperBus\FrontEnd\node_modules\.staging\ng2-data-table-f8e5bea5
npm verb tar unpack C:\Users\jonhs\AppData\Roaming\npm-cache\ng2-data-table\1.0.1\package.tgz
npm verb tar unpacking to D:\Work\Project\TripperBus\TripperBus\FrontEnd\node_modules\.staging\ng2-data-table-f8e5bea5
npm verb gentlyRm don't care about contents; nuking D:\Work\Project\TripperBus\TripperBus\FrontEnd\node_modules\.staging\ng2-data-table-f8e5bea5
npm verb gentlyRm don't care about contents; nuking D:\Work\Project\TripperBus\TripperBus\FrontEnd\node_modules\.staging\ng2-data-table-f8e5bea5\node_modules
npm info lifecycle ng2-data-table@1.0.1~preinstall: ng2-data-table@1.0.1
npm verb unbuild node_modules\ng2-data-table
npm info lifecycle ng2-data-table@1.0.0~preuninstall: ng2-data-table@1.0.0
npm info lifecycle ng2-data-table@1.0.0~uninstall: ng2-data-table@1.0.0
npm verb unbuild rmStuff ng2-data-table@1.0.0 from D:\Work\Project\TripperBus\TripperBus\FrontEnd\node_modules
npm info lifecycle ng2-data-table@1.0.0~postuninstall: ng2-data-table@1.0.0
npm verb gentlyRm don't care about contents; nuking D:\Work\Project\TripperBus\TripperBus\FrontEnd\node_modules\ng2-data-table
npm info linkStuff ng2-data-table@1.0.1
npm verb linkBins ng2-data-table@1.0.1
npm verb linkMans ng2-data-table@1.0.1
npm info lifecycle ng2-data-table@1.0.1~install: ng2-data-table@1.0.1
npm info lifecycle ng2-data-table@1.0.1~postinstall: ng2-data-table@1.0.1
npm verb unlock done using C:\Users\jonhs\AppData\Roaming\npm-cache\_locks\staging-707e5d5003dbcf66.lock for D:\Work\Project\TripperBus\TripperBus\FrontEnd\node_modules\.staging
npm verb saving [ { name: 'ng2-data-table', spec: '^1.0.1', save: 'dependencies' } ]
front-end@0.0.0 D:\Work\Project\TripperBus\TripperBus\FrontEnd
+-- UNMET PEER DEPENDENCY @angular/core@4.1.3
+-- ng2-data-table@1.0.1
`-- UNMET PEER DEPENDENCY tslib@^1.6.1

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm verb notsup SKIPPING OPTIONAL DEPENDENCY: Valid OS:    darwin
npm verb notsup SKIPPING OPTIONAL DEPENDENCY: Valid Arch:  any
npm verb notsup SKIPPING OPTIONAL DEPENDENCY: Actual OS:   win32
npm verb notsup SKIPPING OPTIONAL DEPENDENCY: Actual Arch: x64
npm WARN angular-2-dropdown-multiselect@1.3.2 requires a peer of tslib@^1.6.1 but none was installed.
npm verb
npm verb If you need help, you may report this error at:
npm verb     <https://github.com/npm/npm/issues>
npm WARN ng2-data-table@1.0.1 requires a peer of @angular/core@^2.4.5 but none was installed.
npm verb
npm verb If you need help, you may report this error at:
npm verb     <https://github.com/npm/npm/issues>
npm verb exit [ 0, true ]
npm info ok

D:\Work\Project\TripperBus\TripperBus\FrontEnd>
```

I have updated my angular-cli to version 1.1.0, and Error is same.
How can I fix it?

------------------------------

I think it's related your package.json devDependencies & peerDependencies.

> "devDependencies": {
>     "@angular/common": "^2.4.5",
>     "@angular/compiler": "^2.4.5",
>     "@angular/compiler-cli": "^2.4.5",
>     "@angular/core": "^2.4.5",
> ...
> "peerDependencies": {
>     "@angular/core": "^2.4.5",

**Comment by lar-ang** [comment]
When can I download your fixed version?

**Comment by jdcalkins** [comment]
I published a fix with version 1.0.4. I tested it with angular-cli v1.1.0.

**Comment by lar-ang** [comment]
Building error resolved.
